src/main/java/org/skypro/sky/shop/controller/ShopControllerAdvice.java

```diff
@@ -0,0 +1,17 @@
+package org.skypro.sky.shop.controller;
+
+import org.skypro.sky.shop.model.exception.NoSuchProductException;
+import org.skypro.sky.shop.model.exception.ShopError;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
для RestController-ов